Content.Client/Eye/Blinding/BlindingSystem.cs

```diff
@@ -52,4 +62,9 @@ private void OnBlindShutdown(EntityUid uid, BlindableComponent component, Compon
             _overlayMan.RemoveOverlay(_overlay);
         }
     }
+
+    private void RoundRestartCleanup(RoundRestartCleanupEvent ev)
+    {
+        _lightManager.Enabled = true;
+    }
```

Comment:
Are players not detached on round restart?

Reply:
Apparently not, or this bug wouldnt exist